src/state/vimState.ts

```diff
@@ -42,6 +43,8 @@ export class VimState implements vscode.Disposable {
 
   public editor: vscode.TextEditor;
 
+  public inputMethodSwitcher: InputMethodSwitcher;
```

Comment:
Does this need to be public?

Reply:
I will get all thoese problems fixed. Thanks for your review!

Reply:
Let me know if you have any questions on my comments or it doesn't make any sense . Thanks again for your contribution!